src/lib.rs

```diff
@@ -216,13 +183,25 @@ fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg>) {
 // ------ ------
 
 fn view(model: &Model) -> Vec<Node<Msg>> {
-    nodes![
-        view_header(&model.new_todo_title),
-        IF!(not(model.todos.is_empty()) => vec![
-            view_main(&model.todos, model.selected_todo.as_ref(), model.filter),
-            view_footer(&model.todos, model.filter, &model.base_url),
-        ]),
-    ]
+    if router().current_route().is_default() {
+        vec![div![
+            "page not found ",
+            br![],
+            C!["view"],
+            a![
+                "Click me to find your items! ",
+                attrs! {At::Href => Route::All.to_url()},
+            ]
+        ]]
+    } else {
+        nodes![
+            view_header(&model.new_todo_title),
+            IF!(not(model.todos.is_empty()) => vec![
+                view_main(&model.todos, model.selected_todo.as_ref(), router().current_route()),
```

Comment:
Why we pass `Route` from `router().current_route()` to `view_main`? it looks like we need it only in `view_todo_list`.

Reply:
Yes you are write.

Because my brain is foggy these days :smile:

Reply:
I also made the right change for this